engine/src/main/java/org/terasology/rendering/nui/SortOrderSystem.java

```diff
@@ -103,17 +108,17 @@ public boolean onRelease(float delta, EntityRef target) {
              }
          };
 
-         if (keys.containsKey(Keyboard.Key.LEFT_SHIFT.getId())) {
```

Comment:
This whole method seems like it should be structured using the event system and the built in binding method but that's a topic for a different PR